infrastructure/zk/src/status.ts

```diff
@@ -0,0 +1,210 @@
+import { Command } from 'commander';
+
+import { Pool } from 'pg';
+import { ethers } from 'ethers';
+import { assert } from 'console';
+
+// Postgress connection pool - must be intialized later - as the ENV variables are set later.
+let pool: any = null;
+
+const GETTER_ABI = [
```

Comment:
More of a nit/suggestion, but if you want to hardcode the ABIs, ethers supports the following format

```
const GETTER_ABI = [
    'function getTotalBlocksCommitted() returns (uint256)', 
    'function getTotalBlocksVerified() returns (uint256)'
]
```

Reply:
oh - that's super nice. Thanks !